.release-notes/next-unwrap.md

```diff
@@ -0,0 +1,105 @@
+## Make working with Promise[Promise[Foo]] chains easier
```

Comment:
```suggestion
## Make working with Promise[Promise[B]] chains easier
```

As all the other uses of `Promise[Promise[_]]` use `B` rather than `Foo` might as well be consistent.